src/main/java/com/stripe/model/TransferReversal.java

```diff
@@ -0,0 +1,190 @@
+  /** Get expanded `destinationPaymentRefund`. */
+  public Refund getDestinationPaymentRefundObject() {
+    return (this.destinationPaymentRefund != null)
+        ? this.destinationPaymentRefund.getExpanded()
+        : null;
+  }
+
+  public void setDestinationPaymentRefundObject(Refund expandableObject) {
+    this.destinationPaymentRefund =
+        new ExpandableField<Refund>(expandableObject.getId(), expandableObject);
+  }
+
+  /** Get id of expandable `sourceRefund` object. */
+  public String getSourceRefund() {
+    return (this.sourceRefund != null) ? this.sourceRefund.getId() : null;
+  }
+
+  public void setSourceRefund(String id) {
+    this.sourceRefund = ApiResource.setExpandableFieldId(id, this.sourceRefund);
+  }
+
+  /** Get expanded `sourceRefund`. */
+  public Refund getSourceRefundObject() {
+    return (this.sourceRefund != null) ? this.sourceRefund.getExpanded() : null;
+  }
+
+  public void setSourceRefundObject(Refund expandableObject) {
+    this.sourceRefund = new ExpandableField<Refund>(expandableObject.getId(), expandableObject);
+  }
+
+  /** Get id of expandable `transfer` object. */
+  public String getTransfer() {
+    return (this.transfer != null) ? this.transfer.getId() : null;
+  }
+
+  public void setTransfer(String id) {
+    this.transfer = ApiResource.setExpandableFieldId(id, this.transfer);
+  }
+
+  /** Get expanded `transfer`. */
+  public Transfer getTransferObject() {
+    return (this.transfer != null) ? this.transfer.getExpanded() : null;
+  }
+
+  public void setTransferObject(Transfer expandableObject) {
+    this.transfer = new ExpandableField<Transfer>(expandableObject.getId(), expandableObject);
+  }
+
+  /**
+   * Updates the specified reversal by setting the values of the parameters passed. Any parameters
+   * not provided will be left unchanged.
+   *
+   * <p>This request only accepts metadata and description as arguments.
+   */
+  public TransferReversal update(Map<String, Object> params) throws StripeException {
+    return update(params, (RequestOptions) null);
+  }
+
+  /**
+   * Updates the specified reversal by setting the values of the parameters passed. Any parameters
+   * not provided will be left unchanged.
+   *
+   * <p>This request only accepts metadata and description as arguments.
+   */
+  public TransferReversal update(Map<String, Object> params, RequestOptions options)
+      throws StripeException {
+    String url;
+    if (this.getTransfer() != null) {
+      url =
+          String.format(
+              "%s%s",
+              Stripe.getApiBase(),
+              String.format("/v1/transfers/%s/reversals/%s", this.getTransfer(), this.getId()));
+    } else {
+      throw new InvalidRequestException(
+          "Unable to construct url because [transfer] field(s) are all null",
+          null,
+          null,
+          null,
+          0,
+          null);
+    }
+    return request(ApiResource.RequestMethod.POST, url, params, TransferReversal.class, options);
```

Comment:
12) Update (standard): Instance method to update a sub-resource, relying on ID attributes of the instance, and of its parent resource. There is validation on presence of parent resource ID

Reply:
https://github.com/stripe/stripe-java/blob/2aa5db5c39dc6e91228507798083f373904cc8c0/src/main/java/com/stripe/model/TransferReversal.java#L164-L189